ci/build_wheel_distributed_ucxx.sh

```diff
@@ -4,10 +4,13 @@
 set -euo pipefail
 
 package_dir="python/distributed-ucxx"
+wheel_dir=${RAPIDS_WHEEL_BLD_OUTPUT_DIR}
 
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen "${RAPIDS_CUDA_VERSION}")"
 
 ./ci/build_wheel.sh distributed-ucxx "${package_dir}"
-./ci/validate_wheel.sh "${package_dir}" dist
+mkdir -p "${wheel_dir}"
+cp "${package_dir}/dist"/* "${wheel_dir}/"
+./ci/validate_wheel.sh "${package_dir}" "${wheel_dir}"
```

Comment:
can we make changes here similar to the ones recommended in https://github.com/rapidsai/cudf/pull/18346#pullrequestreview-2737415567?